react-accordion-component/my-app/src/Accordion.js

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react';
+
+export default function Accordion({ topics }) {
+  const [activeIndex, setActiveIndex] = useState(0);
```

Comment:
Looking at how this is used, it's actually not the index that you are holding in your state variable, it's the topic's ID. I suggest you rename this variable to match its usage, because 'index' is misleading.